### ISSUE TITLE
Missing Test dataset in zerowaste-v2-splits

### ISSUE THREAD

**Issue by leoil** [dbash/visda2022-org#1]
Hello, I was using  `configs/daformer/zerowaste_to_zerowastev2_daformer_mit5.py` to train the DAFormer, but  it seems the test dataset is not provided in  zerowaste-v2-trainval.zip file. 
I'm wondering weather i am using the right config and dataset.

The test dir is requied here: 

https://github.com/dbash/visda2022-org/blob/33eb7e2a73069f23abc867739fc8ed60c41aa0da/configs/_base_/datasets/uda_zerowaste_to_zerowastev2_512x512.py#L85

And here is my extracted dataset.
```
./data
├── zerowaste-f
│   ├── test
│   ├── train
│   └── val
└── zerowaste-v2-splits
    ├── train
    └── val
```

**Comment by dbash** [comment]
Hi, thanks for reaching out regarding this. 

Yes, the test set of ZeroWaste-V2 will become publicly available at Phase 2 of the challenge. For now, to evaluate on the validation set, there are two options: 1) temporarily rename the val folder to test or 2) modify the config file to point to the validation set instead of test set:  data_root='data/zerowaste-v2-splits/val'.

Please let us know if that didn't work.
